addons/l10n_fr_fec/wizard/account_fr_fec.py

```diff
@@ -146,7 +146,7 @@ def generate_fec(self):
 
         rows_to_write = [header]
         # INITIAL BALANCE
-        unaffected_earnings_xml_ref = 'equity_unaffected'
+        unaffected_earnings_xml_ref = self.env['account.account'].search([('account_type', '=', 'equity_unaffected'), ('company_id', '=', company.id)])
```

Comment:
Is the variable name still relevant?
If there could be several accounts of type `equity_unaffected` and we only need to know if there is at least one, it might be nice to add a `limit=1` to the search?

Reply:
I agree, changed it :slightly_smiling_face: